baleen/models.py

```diff
@@ -134,14 +136,14 @@ def hash(self):
         """
         return hash_string(self.content)
 
-    def htmlize(self):
+    def htmlize(self, level=RAW):
         """
-        Returns an HTML string of the content of the Post.
-        In the future we may use bleach to do sanitization or other simple
-        sanity checks to ensure that things are going ok, which is why this
-        method stub exists.
+        Returns the content of the Post with html sanitized
+        :param level: the level of sanitizing, default to RAW
+        :return:
         """
-        return self.content
+
```

Comment:
I really like what you've done here, it is extremely sensible. However, I'd ask for one minor change so that this is not a complete passthrough to the sanitize function. What would you think about changing the signature to accept `None` as the argument to level and changing the level argument to sanitize, e.g.:

```python
def htmlize(self, sanitize=None):
    """
    ... 
    
    :param sanitize: str or None, default None 
        Specify a level to sanitize the HTML. If ``None``, then no HTML sanitization will 
        be conducted. Otherwise, specify ``'raw'``, ``'safe'``, or ``'text'`` to sanitize 
        the HTML at different levels. See ref::``sanitize_html`` for more. 
    """
    if sanitize:
        return sanitize_html(html=self.content, level=sanitize)
    return self.content 
```

I think this makes it a bit more understandable to users, what do you think?